src/main/features/testing-support/routes/delete-drafts.ts

```diff
@@ -32,32 +19,13 @@ export default express.Router()
   )
   .post(Paths.deleteDraftsPage.uri,
     ErrorHandling.apply(async (req: express.Request, res: express.Response) => {
-
       const userAuthToken: string = res.locals.user.bearerToken
 
-      const action: string = getAction(req)
-      switch (action) {
-        case 'claim':
-          const draftClaims: Draft<DraftClaim>[] = await getDraft(userAuthToken, 'claim', (value: any): DraftClaim => {
-            return new DraftClaim().deserialize(value)
-          })
-          draftClaims.forEach((async draft => await DraftService.delete(draft, userAuthToken)))
-          break
-        case 'response':
-          const responseDrafts: Draft<ResponseDraft>[] = await getDraft(userAuthToken, 'response', (value: any): ResponseDraft => {
-            return new ResponseDraft().deserialize(value)
-          })
-          responseDrafts.forEach((async draft => await DraftService.delete(draft, userAuthToken)))
-          break
-        case 'ccj':
-          const ccjDraft: Draft<DraftCCJ>[] = await getDraft(userAuthToken, 'ccj', (value: any): DraftCCJ => {
-            return new DraftCCJ().deserialize(value)
-          })
-          ccjDraft.forEach((async draft => await DraftService.delete(draft, userAuthToken)))
-          break
-        default:
-          throw new Error(`Unknown action: ${action}`)
-      }
+      const drafts = await draftService.find(getDraftType(req), '100', userAuthToken, (value) => value)
```

Comment:
Lol, much simpler.
in my defence, the current code required a whole draft for delete :(